code/modules/jobs/job_types/blueshield.dm

```diff
@@ -3,27 +3,30 @@ Blueshield
 */
 
 /datum/job/blueshield
-  title = "Blueshield"
-  flag = BLUESHIELD
-  department_flag = CIVILIAN
-  faction = "Station"
-  total_positions = 1
-  spawn_positions = 1
-  supervisors = "captain and command personnel"
-  selection_color = "#ddddff"
-  req_admin_notify = 1
-  minimal_player_age = 14
+	title = "Blueshield"
+	flag = BLUESHIELD
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "captain and command personnel"
+	selection_color = "#ddddff"
+	req_admin_notify = 1
+	minimal_player_age = 14
+	exp_requirements = 600 //10 hours
+	exp_type = EXP_TYPE_CREW
+	special_notice = "You are not a security officer. Do not do their job."
```

Comment:
Maybe change to Reminder, you are a Bodyguard to Heads of Staff. Not a Security Officer. Do not do their job.

Reply:
Done.